### PR TITLE
qm-pending: add page

### PR DESCRIPTION
**Pull request by Muskan-Git-Code**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**
Reference : #8537

### CODE REVIEW

**Comment by abhisheksrivastava177** [comment]
> * [x]  The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
> * [x]  The page(s) have at most 8 examples.
> * [x]  The page description(s) have links to documentation or a homepage.
> * [x]  The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
> * [x]  The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
> * **Version of the command being documented (if known):**

**Comment by abhisheksrivastava177** [comment]
@Muskan-Git-Code I don't think you need to mention issue in the command version.

**Comment by kbdharun** [comment]
> @Muskan-Git-Code I don't think you need to mention issue in the command version.

I would recommend mentioning issues in the PR description. Like `Reference: #issue number` or `or Ref: #issue number` or whatever you feel convenient. `qm` is a command in which we are getting a lot of PRs so we would link this PR in issue description. 

Also, Whenever a PR closes a issue I would recommend adding `Closes: #no` to the description.

**Comment by abhisheksrivastava177** [comment]
@kbdharun Sure. Quick question : The `Closes: #no` description will close the entire issue, right? Which we should not do in case of qm PRs which only the document a subset of the commands?



> > @Muskan-Git-Code I don't think you need to mention issue in the command version.
> 
> I would recommend mentioning issues in the PR description. Like `Reference: #issue number` or `or Ref: #issue number` or whatever you feel convenient. `qm` is a command in which we are getting a lot of PRs so we would link this PR in issue description.
> 
> Also, Whenever a PR closes a issue I would recommend adding `Closes: #no` to the description.


@kbdharun Sure. Quick question : The `Closes: #no` description will close the entire issue, right? Which we should not do in case of qm PRs which only the document a subset of the commands?

**Comment by kbdharun** [comment]
> @kbdharun Sure. But quick question : The `Closes: #no` description will close the entire issue, right? Which we should not do in case of qm PRs which only document a subset of qm commands?

Yes, It closes the entire issue and it **isn't applicable here**. I mentioned it generally for future reference. We use it for page requests (issues) for individual commands. And let's say there is a Let's document (issue) with 20 commands and 19 were already added. You use `Closes: #issue_no` in that case as with your PRs merge all commands are added.

File: pages/linux/qm-pending.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# qm pending
+
+> Get the virtual machine configuration with both current and pending values.
+> More information: <https://pve.proxmox.com/pve-docs/qm.1.html>.
+
+- Get the virtual machine configuration:
```
Comment (kbdharun) [inline pages/linux/qm-pending.md:6]:
```suggestion
- Get the virtual machine configuration of a specific virtual machine:
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have a suggestion for this page.

**Review by kbdharun — APPROVED** [review]

**Review by abhisheksrivastava177 — APPROVED** [review]

File: pages/linux/qm-pending.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# qm pending
+
+> Get the virtual machine configuration with both current and pending values.
+> More information: <https://pve.proxmox.com/pve-docs/qm.1.html>.
+
+- Get the virtual machine configuration of a specific virtual machine:
+
+`qm pending {{vmid}}`
```
Comment (EmilySeville7cfg) [inline pages/linux/qm-pending.md:8]: To make several PR's consistent take a look at #9110 where `vm_id` is used. 
```suggestion
`qm pending {{vm_id}}`
```

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]

**Review by kbdharun — APPROVED** [review]

**Review by EmilySeville7cfg — APPROVED** [review]

**Comment by EmilySeville7cfg** [comment]
Merged, thanks! ❤️

**Pull request merged**